packages/expo-image/android/src/main/java/expo/modules/image/ExpoImageManager.java

```diff
@@ -40,9 +44,34 @@ public void setSource(ImageView view, @Nullable ReadableMap sourceMap) {
       return;
     }
 
+    RequestOptions options = new RequestOptions();
+
+    // Override the size for local assets. This ensures that
```

Comment:
How (does?) does it work for remote assets? 😕

Reply:
It doesn't. Local assets have a so-called scale-suffix in the file-name (`@3x`) making it possible to who that image by default in it's "real" resolution. This is not possible for remotely loaded assets. For those kinds of images you will have to specify the `width` and `height` in the style or in the `source`.